package.json

```diff
@@ -29,7 +29,7 @@
     "@faststore/sdk": "^1.3.28",
     "@faststore/ui": "^1.3.28",
     "@vtex/gatsby-plugin-nginx": "^1.3.22",
-    "@vtex/gatsby-plugin-thumbor": "^1.3.22",
+    "@vtex/gatsby-plugin-thumbor": "https://pkg.csb.dev/vtex/faststore/commit/47ec99e8/@vtex/gatsby-plugin-thumbor",
```

Comment:
Why are we not using the version number here anymore? What does this mean?

Reply:
It's using this PR: https://github.com/vtex/faststore/pull/1098. After merging it in there we can update the version in here.